### PR TITLE
Deploy operator webhook with operator

### PR DESCRIPTION
**Pull request by pliurh**
Don't merge before https://github.com/openshift/release/pull/5537

### CODE REVIEW

File: bindata/manifests/operator-webhook/server.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: operator-webhook
+    spec:
+      serviceAccountName: operator-webhook-sa
+      nodeSelector:
+        beta.kubernetes.io/os: linux
```
Comment (zshi-redhat) [inline bindata/manifests/operator-webhook/server.yaml]: nit: this is now `kubernetes.io/os`

Comment (pliurh) [reply bindata/manifests/operator-webhook/server.yaml]: As label `beta.kubernetes.io/os` is also used in 4.3, we decide to not change it for this release.

**Pull request merged**